tests/testing/unit/renderer.tsx

```diff
@@ -222,6 +239,19 @@ describe('test renderer', () => {
 			);
 		});
 
+		it('trigger property when there are undefined children in actual render', () => {
+			const WrappedRoot = wrap('div');
+			const WrappedChild = wrap('div');
+			const WrappedConditional = wrap('div');
+			const baseTemplate = assertion(() => w(WrappedRoot, {}, [w(WrappedChild, { onclick: noop })]));
```

Comment:
If you're using wrapped vnodes with the functional API we should be using `v()` and then calling `.tag` from the WrappedNode - `v(WrappedRoot.tag` - alternatively using TSX. I thought there was mention of this in the docs but I can't find it.